Tools/machines/adastra-cines/adastra_warpx.profile.example

```diff
@@ -24,9 +24,9 @@ module load boost/1.83.0-mpi-python3
 # optional: for openPMD support
 module load cray-hdf5-parallel
 export CMAKE_PREFIX_PATH=${SHAREDHOMEDIR}/sw/adastra/gpu/c-blosc-1.21.1:$CMAKE_PREFIX_PATH
-export CMAKE_PREFIX_PATH=${SHAREDHOMEDIR}/sw/adastra/gpu/adios2-2.8.3:$CMAKE_PREFIX_PATH
+export CMAKE_PREFIX_PATH=${SHAREDHOMEDIR}/sw/adastra/gpu/adios2-2.10.2:$CMAKE_PREFIX_PATH
 
-export PATH=${HOME}/sw/adastra/gpu/adios2-2.8.3/bin:${PATH}
+export PATH=${HOME}/sw/adastra/gpu/adios2-2.10.2/bin:${PATH}
```

Comment:
I think that the path here should be changed as follows:
```suggestion
export PATH=${SHAREDHOMEDIR}/sw/adastra/gpu/adios2-2.10.2/bin:${PATH}
```

Reply:
Good catch!